pkg/processor/build/builder.go

```diff
@@ -876,7 +944,7 @@ func (b *Builder) buildProcessorImage() (string, error) {
 		return "", errors.Wrap(err, "Failed to get build args")
 	}
 
-	processorDockerfilePathInStaging, err := b.createProcessorDockerfile()
+	processorDockerfileInfo, err := b.createProcessorDockerfile(b.options.FunctionConfig.Spec.Build.Registry)
```

Comment:
I would make a new variable for this - something like base image registry or something. It may very well be different than the registry here (which is used to store the build artifacts)